docs/Changelog.md

```diff
@@ -3,6 +3,8 @@
 ## 2.0 (in progress)
 - Sandbox for experimental features (#2745)
 - Bugfix for Missing platforms for stops in GTFS import causes a NPE (#2804)
+- Remove extra Djikstra implementations
+- Remove redundant LineStrings in order to save memory (#2795)
 
```

Comment:
Looks good, @gmellemstrand feel free to approve (I am not allowed to by GitHub as PR submitter ) this change and merge PR.